src/CodingStandards/Javascript/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+name: Javascript
+menu: Coding Standards
+---
+
+# Javascript
+
+This guide documents the suggested standards for writing and formatting Javascript code and maintain the same code consistency in all EIGI's client application source code. 
+
+We would be following the `Airbnb` style guide to maintain consistency and maintainability across engineering teams.
+
+_Note:_ `Airbnb`guide assumes you are using [Babel](https://babeljs.io/), and requires that you use [babel-preset-airbnb](https://www.npmjs.com/package/babel-preset-airbnb) or the equivalent. It also assumes you are installing shims/polyfills in your app, with [airbnb-browser-shims](https://www.npmjs.com/package/airbnb-browser-shims) or the equivalent.
+
+## General
```

Comment:
I would like to abstract these out into its own `.mdx` files but every new file added a new section in the left nav bar. Any suggestions?